client/extensions/woocommerce/woocommerce-services/views/shipping-label/label-purchase-modal/rates-step/list.js

```diff
@@ -51,7 +51,8 @@ export const ShippingRates = ( {
 		}
 		const selectedRate = selectedRates[ pckgId ] || '';
 		const packageRates = availableRates[ pckgId ].default.rates;
-		const packageErrors = errors[ pckgId ] || [];
+		// filter out duplicate error messages
+		const packageErrors = errors[ pckgId ].filter( ( item, index ) => errors[ pckgId ].indexOf( item ) === index ) || [];
```

Comment:
Before it was checking that `errors[ pckgId ]` was always an array - now it's assuming it is.
Should this be instead:
```suggestion
		const packageErrors = ( errors[ pckgId ] || [] ).filter( ( item, index ) => errors[ pckgId ].indexOf( item ) === index );
```

Reply:
Great catch, thank you for that!

Reply:
Thank _you_ still looking good after our discussion! 💯